packages/hub/readme.md

```diff
@@ -57,6 +57,20 @@ NODE_ENV=test yarn db:create
 yarn test:ci
 ```
 
+### Troubleshooting Test Errors
+
+If you run into the following error
+
+```shell
+    getaddrinfo ENOTFOUND RequiredInLocalDotenv RequiredInLocalDotenv:5432
+```
+
+Make sure you have a local .env file dedicated to testing, by running the following:
+
+```shell
+cp .env.development.local .env.test.local
```

Comment:
Could we set the necessary details in `.env.test` to avoid this error completely?

Reply:
Good thought, but it will not work unfortunately. Environment variables set outside the executable (and passed to the executable) take precedence over env vars read in from `.env` files.

Reply:
I guess what I'm asking is could we set [these](https://github.com/statechannels/monorepo/blob/hub/assert_test_env_var/packages/hub/.env#L1-L2) to some standard values, that are likely to work on most laptops? Like `localhost` and `5432` or whatever the standard postgres port is?

Reply:
I like that idea. Do you know if port 5432 is standard for postgres on linux and windows?

Even if we set default host and port variables, but `NODE_ENV=development` is passed to jest tests, tests will fail. I spent a bit of time trying to understand why, but didn't get to the bottom of the issue. It seems that jest has a strong stance on NODE_ENV being set to `test`. I see hints about this NODE_ENV in some jest docs and issues. For example: https://jest-bot.github.io/jest/docs/getting-started.html.

Reply:
Filed https://github.com/statechannels/monorepo/issues/693.

Reply:
I think if we just put `5432` in, then that should be fine. I'm hoping that knex will give a resonable error (e.g. "Can't connect to postgres at http://localhost:5432") if that isn't right, which should be enough to point them in the right direction. Might be worth verifying what happens (especially given my previous experience, where it just connected to the default database 🙀 ).

Reply:
Yep, I am onboard with that. Good though on the default db. Tests pass with no `HUB_DB_NAME` defined (and the default db present on the database server).

Reply:
This PR fixes the NODE_ENV issue and adds defaults to `.env.test`: https://github.com/statechannels/monorepo/pull/695